aws_kms_key.tf

```diff
@@ -4,14 +4,14 @@
 
 
 resource "aws_kms_key" "kmsKeyDisabled" {
-  description         = "KMS key 2"
-  is_enabled          = false
+  description = "KMS key 2"
+  is_enabled  = true
 }
 
 resource "aws_kms_key" "kmsKeyRotationDisabled" {
   description         = "KMS key 2"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure rotation for customer created CMKs is enabled**
 
**Vulnerability**
Ensure rotation for customer created CMKs is enabled
 
**Remediation**
(From Console) In the navigation pane, choose Customer managed keys. Choose the alias or key ID of a CMK. Under General configuration, choose the Key rotation tab. (AWS Cli) Execute 'aws kms enable-key-rotation --key-id <KMS_KEY_ID>'

source: accurics-